subjects/scores/README.md

```diff
@@ -20,17 +20,8 @@ You will need these:
 | J, X                         |   8   |
 | Q, Z                         |  10   |
 
-### Notions
-
-- [patterns](https://doc.rust-lang.org/book/ch18-00-patterns.html)
-
 ### Expected functions
```

Comment:
If we decide to remove the function here we need to remove the "### Expected Functions" as well

Reply:
Replaced my comment, as I had accidentally started a review previously.

The following is below:
> You'll need to work out the function signature for yourself.